authd_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-grpc v1.4.0
-// - protoc             v5.27.1
+// - protoc-gen-go-grpc v1.5.1
+// - protoc             v4.25.1
```

Comment:
Mh, why protoc is older now?

Reply:
I noticed it, but I think that was a mistake, I just downloaded straight from GH a recent release. Something probably cheated at some points (even the others protoc definition were 3.)